arm-oraclelinux-wls-cluster/src/main/arm/nestedtemplates/clusterTemplate.json

```diff
@@ -244,6 +244,21 @@
                             "direction": "Inbound",
                             "destinationPortRanges": [ "7005" ]
                         }
+                    },
+                    {
+                        "name": "WebLogiManagedChannelPortsDenied",
```

Comment:
Any reason why this has `WebLogi` instead of `WebLogic`?

Reply:
I missed that spot, and will fix the type, thanks